_posts/2022-10-18-trino-summit-2022-teaser-2.md

```diff
@@ -0,0 +1,141 @@
+our journey.
+
+JaeChang Song, Data Engineer at SKTelecom and Trino/Iceberg Contributor
+
+Jennifer OH, Data Engineer at SKTelecom
+
+### Elevating Data Fabric to Data Mesh: solving data needs in hybrid data lakes
+
+At Comcast, we have long had a complex hybrid data lakes that consists of
+data lakes in on-prem and multiple cloud environments. Comcast uses Trino to
+bridge the data in these environments using an architecture we call Data Fabric.
+Data Fabric is an abstraction layer that uses an internally built connector that
+connects to multiple instances of Trino. This enables us to query across all
+of these environments from a single Trino instance.
+
+In recent years, emerging architectures like Data Mesh have nicely complemented
+the goals we have been building to for years. While we have effectively 
+implemented some aspects of a Data Mesh, there are still core tenants that
+cannot be addressed by Trino alone. This is the journey we are on at Comcast,
+and we like to share our experience so far, challenges we overcame, and
+the ones yet to be resolved. Data abstraction, availability, movement, and
+governance are the various topics we will touch upon in this session.
+
+Sajumon Joseph, Sr Principal Architect
+Pavan Madhineni, Sr. Manager; Product Development Engineering
+
+### Trino at Quora: Speed, Cost, Reliability Challenges and Tips
+
+Trino has become an essential part of Quora’s tech stack and a major component
+of our A/B testing framework that powers our decision-making on the product.
+Trino has brought a lot of advantages to us. However, at Quora’s scale, we face
+cost, speed, and reliability challenges when operating Trino.
+
+In this session, we will talk about how we resolve the challenges. Some
+approaches are: auto-scale Trino clusters, experiment with different cluster and
+JVM configurations, and instance types, build checkers to detect slow workers
+and inefficient queries, and set up extensive monitoring.
+
+Yifan Pan, Software Engineer of Data Infrastructure Team at Quora; 
+Administrator/Primary Owner of Trino infrastructure at Quora
+
+### How we use Trino to analyze our Product-led Growth (PLG) user activation funnel
+
+Being a PLG company, we must track and analyze every action our users perform
+within the product to remove friction and maximize usage and satisfaction. To
+understand how effectively and quickly users become educated and then active in
+the product, we had to instrument the user journey from signup to the Aha moment
+and beyond.
+
+There are many tools on the market that can be used to analyze user behavior,
+but none met our needs. In this session you will learn how we built a data
+architecture to collect, model, and enrich user behavior events to optimize
+Trino query performance that accelerated our ability to understand and improve
+user conversion rates.
+
+Roy Hasson, Head of Product at Upsolver
+
+## Conclusion
+
+Stay tuned for new developments in upcoming blog posts, don't forget to
```

Comment:
Somehow say that the summit will be great and we will forward to seeing everyone.. and then the rest